frontend/src/metabase/nav/components/search/SearchButton/SearchButton.tsx

```diff
@@ -14,21 +14,36 @@ export const SearchButton = () => {
   }, [setVisualState]);
 
   return (
-    <Tooltip label={`${t`Search...`} (${METAKEY}+k)`}>
-      <Button
-        h="36px"
-        w="240px"
-        leftIcon={<Icon name="search" />}
-        onClick={handleClick}
-        // TODO: Adjust this with Mantine V7
-        styles={{
-          inner: {
-            justifyContent: "start",
-          },
-        }}
-      >
-        {t`Search`}
-      </Button>
-    </Tooltip>
+    <>
+      <MediaQuery smallerThan="sm" styles={{ display: "none" }}>
+        <Tooltip label={`${t`Search...`} (${METAKEY}+k)`}>
+          <Button
+            h="36px"
+            w="240px"
+            leftIcon={<Icon name="search" />}
+            onClick={handleClick}
+            // TODO: Adjust this with Mantine V7
+            styles={{
+              inner: {
+                justifyContent: "start",
+              },
+            }}
+            aria-label="Search"
+          >
+            {t`Search`}
+          </Button>
+        </Tooltip>
+      </MediaQuery>
+      <MediaQuery largerThan="sm" styles={{ display: "none" }}>
+        <Button
+          h="36px"
+          leftIcon={<Icon name="search" />}
+          variant="subtle"
+          onClick={handleClick}
+          color="text-medium"
+          aria-label="Search"
+        />
+      </MediaQuery>
```

Comment:
I think it would be more performant and more in line with [the future of styles at metabase](https://www.notion.so/metabase/100-Align-approach-to-Mantine-styles-customization-1066f77c43694c36a6e90a3d2374b731?pvs=4#96398539d9764e419641609cc9828c3c) to give these css classes with media queries instead of introducing the media query component.  WDYT?